advanced/server-extension/static/index.html

```diff
@@ -0,0 +1 @@
+This content is served from the jlab_ext_example server extension.
```

Comment:
Should this be a full HTML document? (doctype and html tags)

Reply:
done